packages/plugin-reconstruction/src/index.spec.ts

```diff
@@ -0,0 +1,175 @@
+        },
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    expect(getHTML()).toContain("<p>5</p>");
+  });
+
+  test("custom starting value renders correctly", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        starting_value: 0.9,
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    expect(getHTML()).toContain("<p>9</p>");
+  });
+
+  test("default increment key works", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        step_size: 0.1,
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    pressKey("h");
+    expect(getHTML()).toContain("<p>6</p>");
+    pressKey("h");
+    expect(getHTML()).toContain("<p>7</p>");
+  });
+
+  test("default decrement key works", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        step_size: 0.1,
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    pressKey("g");
+    expect(getHTML()).toContain("<p>4</p>");
+    pressKey("g");
+    expect(getHTML()).toContain("<p>3</p>");
+  });
+
+  test("custom increment key works", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        key_increase: "a",
+        step_size: 0.1,
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    pressKey("a");
+    expect(getHTML()).toContain("<p>6</p>");
+    pressKey("a");
+    expect(getHTML()).toContain("<p>7</p>");
+    pressKey("h");
+    expect(getHTML()).toContain("<p>7</p>");
+  });
+
+  test("custom decrement key works", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        key_decrease: "a",
+        step_size: 0.1,
+      },
+    ];
+
+    const { getHTML } = await startTimeline(timeline);
+
+    pressKey("a");
+    expect(getHTML()).toContain("<p>4</p>");
+    pressKey("a");
+    expect(getHTML()).toContain("<p>3</p>");
+    pressKey("g");
+    expect(getHTML()).toContain("<p>3</p>");
+  });
+
+  test("custom button label works", async () => {
+    let timeline = [
+      {
+        type: reconstruction,
+        stim_function: function (val) {
+          return `<p>${Math.round(val * 10)}</p>`;
+        },
+        button_label: "foo",
+      },
+    ];
+
+    const { jsPsych } = await startTimeline(timeline);
+
+    expect(jsPsych.getDisplayElement().querySelector("button").innerHTML).toContain("foo");
```

Comment:
There's also
```suggestion
    const { displayElement } = await startTimeline(timeline);

    expect(displayElement.querySelector("button").innerHTML).toContain("foo");
```
(for what it's worth)

Reply:
Missed this option. Thanks!